src/player.js

```diff
@@ -917,6 +928,25 @@ export default class Player extends FakeEventTarget {
     this._loadingMedia = loading;
   }
 
+  /**
+   * getter for _activeTextCues
+   * @returns {Array<any>} returns the active text cues array
+   * @public
+   */
+  get activeTextCues(): Array<any> {
+    return this._activeTextCues;
```

Comment:
No need for setter/getter, use the `TEXT_CUE_CHANGED` event to update the current active displayed text track cues.